assets/js/editor.js

```diff
@@ -628,8 +627,57 @@
           title: 'Plein écran'
         }
       ]
+    })
+
+    if (smdeUniqueContent != null && localStorage['smde_' + mdeUniqueKey] !== textarea.value) {
+      const $alertbox = $('<div class="alert-box info"></div>')
+
+      const $hide = $("<a>Masquer l'alerte</a>")
+
+      $hide.attr({
+        href: 'javascript:void(0)',
+        class: 'close-alert-box ico-after cross white',
+        title: "Masqer l'alerte (Echap: Ferme l'alerte)"
```

Comment:
```suggestion
        title: "Masquer l'alerte (raccourci clavier: Echap)"
```

Typo + À mon avis pas besoin de préciser pour le échap

Reply:
corrigé